apps/cave/components/Marketplace/Main/UseMarkeplaceState.ts

```diff
@@ -41,12 +40,7 @@ export const useMarketplaceDashbord = () => {
   const [sort, setSort] = useState<NftSort>({ sort: 'REDEEM_DATE', order: 'ASC' })
   const sortFunction = sort ? positionSorter.data?.[sort.sort][sort.order] : () => 0
 
-  const [stakeFilters, setStakeFilters] = useState([
-    StakePoolFilterEnum['45_DAYS'],
-    StakePoolFilterEnum['180_DAYS'],
-    StakePoolFilterEnum['180_DAYS'],
-    StakePoolFilterEnum['360_DAYS'],
-  ])
+  const [stakeFilters, setStakeFilters] = useState<StakingPool[]>([...stakingPools])
```

Comment:
need to spread to a new array?  
```suggestion
  const [stakeFilters, setStakeFilters] = useState<StakingPool[]>(stakingPools)
```